File: fixtures/php/src/Calc.php
<?php

class Calc
{
    public function add($a, $b)
    {
        return $a + $b;
    }

    public function divide($a, $b)
    {
        if ($b === 0) {
            throw new InvalidArgumentException('divide by zero');
        }
        return $a / $b;
    }
}

File: fixtures/spring/src/main/java/com/example/Calc.java
package com.example;

public class Calc {
  public int add(int a, int b) {
    return a + b;
  }

  public int divide(int a, int b) {
    if (b == 0) throw new IllegalArgumentException("divide by zero");
    return a / b;
  }
}
